opentimelineview/timeline_widget.py

```diff
@@ -145,24 +148,55 @@ def _set_labels_timecode(self):
             )
         )
 
+    def _set_labels_effects(self):
+        if isinstance(self.item, otio.core.Item):
+            if self.item.effects:
+                self.source_fx_label.setHtml(
+                    "<div style='background:rgba(255,255,255,100%);'>FX</div>"
+                )
+            else:
+                self.source_fx_label.setPlainText('')
+
     def _set_labels(self):
         self._set_labels_rational_time()
         self.source_name_label.setText('PLACEHOLDER')
         self._position_labels()
 
     def _set_tooltip(self):
-        self.setToolTip(self.item.name)
+        toolTipStr = self.item.name
+
+        if isinstance(self.item, otio.schema.Clip) or \
+           isinstance(self.item, otio.schema.Stack):
+            duration = "%0.2f" % otio.opentime.to_seconds(
+                                self.timeline_range.duration
+                                )
+
+            toolTipStr = toolTipStr + "\nLength (secs): " + str(duration)
+
+        if isinstance(self.item, otio.core.Item) and self.item.effects:
+            if len(self.item.effects) == 1:
+                toolTipStr = toolTipStr + \
+                            "\n" + str(len(self.item.effects)) + " effect"
```

Comment:
Might be more scannable if you start with the word effects, then you also don't need the branch.  So the line is:

`"\nEffects: {}".format(len(self.item.effects))`